### PR TITLE
Task/get current user recipes

### PR DESCRIPTION
**Pull request by Abrafaks**

### CODE REVIEW

File: src/controllers/recipe.controller.ts [hunk 1]
```diff
@@ -87,6 +87,24 @@ export class RecipeController {
...
+  public async readUserRecipesById(
+    req: Request,
+    res: Response
+  ): Promise<Response> {
+    try {
+      const { id: readUserId } = matchedData(req);
+
+      const recipes = await recipeService.readRecipeByUserId(readUserId);
+
+      if (!recipes) {
```
Comment (pawelrozumek) [inline src/controllers/recipe.controller.ts:100]: You can do this a bit cleaner when you move constants, if statement and return statement outside trycatch block. That way later on you can easyly add another pice of code witout need to refactor.

File: src/controllers/user.controller.ts [hunk 2]
```diff
@@ -11,6 +11,22 @@ interface UserAuthData {
...
+    try {
+      const { _id } = req.user!;
+
+      const users = await userService.getUsers();
+
+      if (!users) {
+        res.status(StatusCodes.BAD_REQUEST).send();
+      }
+
+      return res.send({ users });
```
Comment (MoodupDev) [inline src/controllers/user.controller.ts]: What is the result here? Is it `{[{...}, {...}, {...}]}` or `[{...}, {...}, {...}]` (should be the second one)

File: src/routes/user.routes.ts [hunk 3]
```diff
@@ -6,6 +6,35 @@ import { validate } from "./middleware/validators/validator";
 
 const router = express.Router();
 
+/**
+ * @swagger
+ * /auth/login:
```
Comment (MoodupDev) [inline src/routes/user.routes.ts]: Wrong documentation

File: src/routes/recipe.routes.ts [hunk 4]
```diff
@@ -133,6 +133,46 @@ router.get(
   recipeController.readRecipeById
 );
 
+/**
```
Comment (pawelrozumek) [inline src/routes/recipe.routes.ts:136]: later on consider moving this documentation to separate files. This things stat po pile up quickly and make router a bit difficult to modify

**Review by pawelrozumek — APPROVED** [review]

**Review by MoodupDev — APPROVED** [review]

File: src/routes/user.routes.ts (hunk 3 above)
Comment (pawelrozumek) [inline src/routes/user.routes.ts]: wrong route

**Review by pawelrozumek — APPROVED** [review]

File: src/routes/recipe.routes.ts (hunk 4 above)
Comment (MoodupDev) [reply src/routes/recipe.routes.ts:136]: I would leave it as it is. This is JSDoc style where we're documenting some method directly above it

File: src/controllers/recipe.controller.ts (hunk 1 above)
Comment (Abrafaks) [reply src/controllers/recipe.controller.ts:100]: If I do it, then try{} block would be empty

**Pull request merged**